content/millennium/r4/proprietary-codes-and-systems.md

```diff
@@ -199,7 +200,7 @@ The Appointment Resource supports proprietary codes for:
 
 * Appointment.participant.type codes are maintained in [Code Set 14250 Scheduling Resource Roles](#code-set-14250-scheduling-resource-roles)
 
-### List of Code Sets
+#### List of Code Sets
```

Comment:
Is there a reason we changed this header?

Reply:
So we can hotlink to this label with line 103 above. This is how it's done in other parts of the code when referring to an appropriate code set. Since this extension works with all code sets, I thought it would be appropriate to provide a link to the top of the code set list.

Reply:
[Example of an implemented hotlink tag](https://github.com/cerner/fhir.cerner.com/pull/438/files/413b26e3fd6d98090bd39b111a8579347609b75b#diff-6fdcb00b5b1b6edd51b687794bea6b95R205)

Reply:
it doesn't make sense to change the h3 here to h4, can we find a way to hyperlink without downgrading?

Reply:
Upon further investigation, this link works on the live version of the site as-is: https://fhir.cerner.com/millennium/r4/proprietary-codes-and-systems/#list-of-code-sets

[Therefore, I have reverted this change as I can hotlink to the tag #list-of-code-sets without having to change the markup language at all.](https://github.com/cerner/fhir.cerner.com/pull/438/commits/529555eeb8a44550552c9fa61088d1c27530179f)